perfkitbenchmarker/pkb.py

```diff
@@ -147,6 +147,14 @@
                      'all disks together.',
                      lower_bound=1)
 
+# support for proxy
+flags.DEFINE_string('http_proxy', '',
+                    'Specify a proxy for HTTP in the form [user:passwd@]proxy.server:port.')
```

Comment:
I had meant adding an explanation to the flags so that people looking at the help text know about it, or a note in the README. But it's probably reasonably clear from context.